responsive-web-design-piano/css/style02.css

```diff
@@ -0,0 +1,26 @@
+/* box model reset */
+html {
+    box-sizing: border-box;
+}
+
+
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+
+/* 
+    Let's start styling and giving form to the piano. Add the id
```

Comment:
Let's start styling and giving form to the piano. 
Create a new rule using the `#piano` `id` selector
and give it a `width` of `992px`.
Here's an example that targets `#guitar` and sets its width:
    ...
        #guitar {
            width: 600px;
        }
     ```